public/js/report.js

```diff
@@ -287,11 +287,24 @@ var Kablammo = (function () {
             .on('mouseenter', null)
```

Comment:
Can you move Kablammo react component to kablammo.js, and appropriately require and use it in report.js?